application/templates/cronjob.yaml

```diff
@@ -43,11 +43,12 @@ spec:
           {{- end }}
           containers:
           - name: {{ $name }}
-            {{- if $job.image.digest }}
-            image: "{{ $job.image.repository }}@{{ $job.image.digest }}"
-            {{- else if $job.image.tag }}
-            image: "{{ $job.image.repository }}:{{ $job.image.tag }}"
-            {{- end }}
+
+            {{- $image := required (print "Undefined image repo for container '" $name "'") $job.image.repository }}
```

Comment:
```suggestion
            {{- $image := required (print "Undefined image repo for cronjob container '" $name "'") $job.image.repository }}
```

Reply:
I find this redundant, as Helm will display the template file where the error was thrown already. Here's an example from a test chart, where I can `helm template --debug .` twice, with errors through in different templates, but with the same error message:

```
install.go:194: [debug] Original chart version: ""
install.go:211: [debug] CHART PATH: /tmp/helm-test


Error: execution error at (test-chart/templates/cronjob.yaml:13:16): This is a failure message
helm.go:84: [debug] execution error at (test-chart/templates/cronjob.yaml:13:16): This is a failure message
```

And then in a Deployment template:

```
install.go:194: [debug] Original chart version: ""
install.go:211: [debug] CHART PATH: /tmp/helm-test


Error: execution error at (test-chart/templates/deployment.yaml:17:12): This is a failure message
helm.go:84: [debug] execution error at (test-chart/templates/deployment.yaml:17:12): This is a failure message
```

What do you think?

Reply:
I was not sure, looks ok to me :)